infra/data-pipeline.yml

```diff
@@ -56,6 +56,34 @@ tasks:
       owner: release-mgmt-analysis@mozilla.com
       source: https://github.com/mozilla/bugbug/raw/master/data-pipeline.yml
 
+  - ID: rollback-test-task
+    created: {$fromNow: ''}
+    deadline: {$fromNow: '25 hours'}
+    provisionerId: aws-provisioner-v1
+    workerType: releng-svc
+    dependencies:
+      - bugs-retrieval
+    payload:
+      maxRunTime: 3600
+      image: python
```

Comment:
Probably better to use the bugbug base image, so we don't have to install the requirements (and we don't have to define yet another image)